lib/text/vtt_text_parser.js

```diff
@@ -380,11 +380,9 @@ shaka.text.VttTextParser = class {
     let end = VttTextParser.parseTime_(parser);
 
     if (start == null || expect == null || end == null) {
-      throw new shaka.util.Error(
-          shaka.util.Error.Severity.CRITICAL,
-          shaka.util.Error.Category.TEXT,
-          shaka.util.Error.Code.INVALID_TEXT_CUE,
-          'Could not parse cue time range in WebVTT');
+      shaka.log.warning(
```

Comment:
Please make this `shaka.log.alwaysWarn`, so the warning is kept in production builds.

Reply:
Done!